pkg/operator/resources/operator/operator.go

```diff
@@ -54,7 +54,12 @@ func getClusterPolicyRules() []rbacv1.PolicyRule {
 				"clusterroles",
 			},
 			Verbs: []string{
-				"*",
```

Comment:
@mhenriks Apparently we have been using `escalate` for a while, which means the operator
can create clusterroles with more rights than it has
https://kubernetes.io/docs/concepts/security/rbac-good-practices/#escalate-verb

Reply:
Yikes!